build.sh

```diff
@@ -1,5 +1,28 @@
 #!/bin/bash
-# python3 -m pip install --ignore-installed --prefix $PWD/pypkgs -r requirements.txt
-# python3 -m pip install --prefix $PWD/pypkgs $PWD/custom_plugin
-# python3 -m pip install --prefix $PWD/pypkgs $PWD/computational_macros
-PYTHONPATH=$PWD/pypkgs/lib/python3.9/site-packages:$PWD/custom_plugin:$PWD/computational_macros:$PYTHONPATH python3 build.py
+set -x
+if [ ${INSTALLDEPS:-0} -gt 0 ]; then
+    python3 -m pip install --ignore-installed --prefix $PWD/pypkgs -r requirements.txt
```

Comment:
`shellcheck build.sh`  shows several warnings such using  double quotes for all variables like "$PWD"